Doc/Tutorial/chapter_entrez.tex

```diff
@@ -261,14 +263,13 @@ \section{ESummary: Retrieving summaries from primary IDs}
 \begin{verbatim}
 >>> from Bio import Entrez
 >>> Entrez.email = "A.N.Other@example.com"     # Always tell NCBI who you are
->>> handle = Entrez.esummary(db="journals", id="30367")
+>>> handle = Entrez.esummary(db="nlmcatalog", term="[journal]", id="101660833")
 >>> record = Entrez.read(handle)
->>> record[0]["Id"]
-'30367'
->>> record[0]["Title"]
-'Computational biology and chemistry'
->>> record[0]["Publisher"]
-'Pergamon,'
+>>> info = record[0]['TitleMainList'][0]
+>>> print("Journal info\nid: {}\nTitle: {}".format(record[0]["Id"], info["Title"]))
```

Comment:
I prefer .format(), and find it very readable, not necessarily better than directly looking at the field values, that said...
My purpose was making the code correct not explanatory. The way the records are now returned is different and within multiple layers of lists and dicts. An explanation of how to find what you need would be helpful, but I am targeting the Jupyter notebooks project when considering improving the quality of the content.